src/test/shared/sam/cli/samCliBuild.test.ts

```diff
@@ -65,11 +65,12 @@ describe('SamCliBuildInvocation', async () => {
             assert.strictEqual(args[0], 'build')
             assert.strictEqual(args[3], '--template')
             assert.strictEqual(args[5], '--base-dir')
-
             // `extraArgs` are appended to the end.
```

Comment:
the blank line above was intentional, to separate this block of code from the above. Otherwise it is not clear that the single-line comment decorates the block rather than the single line of code below it.

Reply:
Fair, I don't think I did this consciously (had to add some muck to the call just to make sure I knew the correct order and this was part of that "cleanup")